dist/mapdc.js

```diff
@@ -79966,7 +79966,7 @@ var ScrollZoomHandler = function (_BaseHandler2) {
   }, {
     key: "_wheelZoom",
     value: function _wheelZoom(doFullRender, delta, e) {
-      if (this._chart.shiftToZoom() && e && !e.shiftKey) {
+      if (this._chart.shiftToZoom() && (!e || !e.shiftKey)) {
```

Comment:
The logic here was slightly wrong - we can fire _wheelZoom events from a timeout which means there is no event attached.

So we want to bow out if it's shiftToZoom AND there's no event (it's the timeout - ignore it) OR we have an event but w/o the shift key.